app/controllers/dashboard/events_controller.rb

```diff
@@ -12,4 +12,6 @@ def permitted_params
   def respond_with_created_resource
     respond_with resource, location: -> { request.headers["Referer"] || dashboard_root_path }
   end
+
+  def prepare_breadcrumbs; end
```

Comment:
What does this do?

Reply:
It skips rendering breadcrumbs because we don't need breadcrumbs in this controller.